tailwind.button.js

```diff
@@ -0,0 +1,206 @@
+const plugin = require("tailwindcss/plugin");
+
+// TODO refactor with theme references after removing SASS/Bootstrap
+// https://tailwindcss.com/docs/theme#referencing-other-values
+
+const btnDefault = {
```

Comment:
Is there a way to specify this in real CSS rather than JS? (e.g. const btnDefault = require("btnDefault.twcss" or something?)

Reply:
We could add it to the sass files using `@layer components` and `@apply/theme()` 

```
@layer components {
  .btn {
    background-color: theme("colors.white");
    border-radius: theme("borderRadius.lg");
    padding: theme("spacing.6");
    box-shadow: theme("boxShadow.xl");
  }
}
```
this solution is a little worse because:
* no prefix added automatically
* have to wait for sass to load when doing changes
* no intellsense
* have to add to both main.scss and buyer-guide.scss

The tailwind plugin system uses CSS in JS but I think the only way we could import css files like that is if we included css modules in our build. Maybe it would be overkill if we used CSS modules for just this part?

Reply:
A few button stylings also includes styling outside of our `tailwind.config.js` so we would have to get some more design feedback if we did it use the theme variables too

https://tailwindcss.com/docs/functions-and-directives

Reply:
Hm, yeah the first one is pretty worse, and the second isn't very appealing: if we need to create an additional transpile step, let's not do that for now and first get tailwind working at all, before making it "cleaner".